adapters/logstash/logstash.go

```diff
@@ -0,0 +1,63 @@
+package logstash
+
+import (
+	"encoding/json"
+	"errors"
+	"io"
+	"net"
+	"os"
+
+	"github.com/gliderlabs/logspout/router"
+)
+
+var hostname string
+
+func init() {
+	router.AdapterFactories.Register(NewLogstashAdapter, "logstash")
+
+	hostname, _ = os.Hostname()
+}
+
+// LogstashAdapter is an adapter that streams UPD JSON to Logstash.
+type LogstashAdapter struct {
+	conn  net.Conn
+	route *router.Route
+}
+
+// NewLogstashAdapter creates a LogstashAdapter with UDP transport.
+func NewLogstashAdapter(route *router.Route) (router.LogAdapter, error) {
+	transport, found := router.AdapterTransports.Lookup(route.AdapterTransport("udp"))
```

Comment:
Okay. Your comments suggest otherwise, so maybe consider removing specific mention of UDP in them.